app/controllers/notes_controller.rb

```diff
@@ -46,6 +46,27 @@ def index
 		render 'index', formats: [:html]
   end
 
+  def all_notes
```

Comment:
Okay, so instead of creating a new route you should reuse the `index` route, but use a `respond_to` block to distinguish between the format being html vs. json.

For example: 

https://github.com/MarkUsProject/Markus/blob/9f4e84326e823670e2e1fcb009f7fc6b75ef4555/app/controllers/students_controller.rb#L8-L38

In the tests, you can use the `index` route but then pass `format: :js` to the method that makes the request to specify the format; look out for examples of this in the existing tests.